content/blog/api-console-for-data-services-cloud-console.md

```diff
@@ -0,0 +1,200 @@
+      "sourceIpAddress": "fleet-gql-data-graph:4000",
+      "state": "PermissionDenied",
+      "taskId": "",
+      "uniqueId": "audit.events+0+3936",
+      "userEmail": "mandy.shen@hpe.com",
+      "version": 1
+    },
+    {
+      "associatedResource": {
+        "id": "/api/v1/host-initiators?filter=editStatus%20in%20(Update_In_Progress,Update_Success,Update_Failed,Not_Applicable,Delete_Failed)",
+        "name": "",
+        "type": ""
+      },
+      "code": "Unauthorized privilege",
+      "contextId": "",
+      "customerId": "e8a4da2eefc311eba02cb603422bc0a0",
+      "id": "f0d2c4c6-d859-42f3-ae4b-60f8d3d2d89d",
+      "loggedAt": "2021-11-10T04:09:03Z",
+      "message": "Unauthorized user access",
+      "occurredAt": "2021-11-10T04:09:03Z",
+      "permission": "data-services.host-initiator.read",
+      "scope": "",
+      "source": "/api/v1/host-initiators?filter=editStatus%20in%20(Update_In_Progress,Update_Success,Update_Failed,Not_Applicable,Delete_Failed)",
+      "sourceIpAddress": "fleet-gql-data-graph:4000",
+      "state": "PermissionDenied",
+      "taskId": "",
+      "uniqueId": "audit.events+2+3975",
+      "userEmail": "mandy.shen@hpe.com",
+      "version": 1
+    },
+    
+    .... snippet ....
+    
+      {
+      "associatedResource": {
+        "id": "/api/v1/storage-systems/volumes?filter=isSystemVolume%20eq%20false\u0026limit=1\u0026offset=0",
+        "name": "",
+        "type": ""
+      },
+      "code": "Unauthorized privilege",
+      "contextId": "",
+      "customerId": "e8a4da2eefc311eba02cb603422bc0a0",
+      "id": "86988487-f7f4-403e-b33e-e15abfcd568a",
+      "loggedAt": "2021-08-19T15:38:49Z",
+      "message": "Unauthorized user access",
+      "occurredAt": "2021-08-19T15:38:49Z",
+      "permission": "data-services.volume.read",
+      "scope": "",
+      "source": "/api/v1/storage-systems/volumes?filter=isSystemVolume%20eq%20false\u0026limit=1\u0026offset=0",
+      "sourceIpAddress": "scalpha-app.qa.cds.hpe.com",
+      "state": "PermissionDenied",
+      "taskId": "",
+      "uniqueId": "audit.events+6+3057",
+      "userEmail": "matt.haron@hpe.com",
+      "version": ""
+    }
+  ],
+  "total": 123,
+  "numItems": 123,
+  "pageLimit": 500,
+  "pageOffset": 0
+}
+```
+
+The recommended tool at this moment of time to experiment with the REST API for DSCC is the Postman which is downloadable from the [Postman website](https://www.postman.com/downloads/). The postman is a versatile tool, where anyone can use copy the access token (or better to use the client-id and client-secret) from the API Gateway and issue a REST API request without typing a lot of code, test the parameters and format the responses.
+
+So this blog is a great example on how to obtain the access-token and experiment with the DSCC API. Please take a look at the next blog on getting the access token programmatically to enable any client-application using any familiar programming or scripting language.
```

Comment:
Remove "So". Change the beginning to "This blog post gives you a great example on how to obtain the access token and experiment with the DSCC API."